src/components/ec-blog-item/ec-blog-item.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div>
+    <p>{{ title }}</p>
+    <p>{{ shortDescription }}</p>
+    <p>{{ author }}</p>
+    <p>{{ category }}</p> hello
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'EcBlogItem',
+  props: {
+    title: { type: String },
+    shortDescription: { type: String },
+    author: { type: String },
+    category: { type: String },
+  },
+};
+</script>
+<style>
+p {
+  font-size: 2em;
+  text-align: center;
```

Comment:
We use tailwind, so you can just replace this with `@apply tw-text-center;` You can see the full list of all Tailwind utility classes here: https://chameleon.ebury.now.sh/?path=/docs/css-tailwind--all.